### DISCUSSION TITLE
Insecure Dupont Jumper Leads

### DISCUSSION THREAD

**Discussion by jimford-sudo** [arendst/Tasmota#17886] [Q&A]
Not strictly Tasmota related, but as many users will use dupont jumper leads I thought that this would be a good place to ask.

I find that many times a dupont jumper lead will appear to be secure when putting on, but will subsequently simply fall off.

Do dupont jumper leads vary greatly in quality and I have simply got a bad lot, and are reliable ones available?

Jim

**Comment by lalo-uy** [comment]
China cheape ones happened to fall off often.
You can always release the plastic cover and tighten them with a pair of
pliers.
Not too mutch or the peg will never came in again.

**Reply by sfromis** [reply]
Sure, dupont wires vary in quality, as does the other half on the board. Such wires cannot be recommended for permanent wiring, just not reliable enough. While you can likely pay for a better mechanical quality, it is still just not good for more than a temporary test build, unless reliability is not important...
